src/lib.rs

```diff
@@ -164,6 +164,71 @@ where
     thread_rng().random()
 }
 
+/// Generate a random value in the given range using the thread-local random number generator.
+///
+/// This function is a shortcut for [`thread_rng().gen_range(range)`](Rng::gen_range).
+///
+/// # Examples
+///
+/// ```
+/// let x = rand::range(1u8..=100);
+/// println!("{}", x);
+///
+/// let y = rand::range(0f32..=1e9);
+/// println!("{}", y);
+/// ```
+///
+/// If you're calling `range()` in a loop, caching the generator as in the
+/// following example can increase performance.
+///
+/// ```
+/// use rand::Rng;
+///
+/// let mut v = vec![1, 2, 3];
+///
+/// for x in v.iter_mut() {
+///     *x = rand::range(1..=3)
```

Comment:
I doubt this will improve performance. Unless you mean to cache the distribution with `Uniform::try_from`?